Documentation/ceph-quickstart.md

```diff
@@ -24,6 +24,7 @@ If you are using `dataDirHostPath` to persist rook data on kubernetes hosts, mak
 If you're feeling lucky, a simple Rook cluster can be created with the following kubectl commands. For the more detailed install, skip to the next section to [deploy the Rook operator](#deploy-the-rook-operator).
 ```
 cd cluster/examples/kubernetes/ceph
+kubectl create -f operator-common.yaml
```

Comment:
Where is this file? As far as I can tell, it doesn't exist anywhere.

Reply:
@jmwilkinson it ended up being called `common.yaml`

Reply:
Ah, so that is what is referenced in the current docs then? That makes sense.